docs/sql_reference/src/asciidoc/_chapters/olap_functions.adoc

```diff
@@ -260,7 +260,7 @@ and <<interval_value_expressions,Interval Value Expressions>>.
 
 * `_inline-window-specification_`
 +
-specifies_the_window_over_which_the_avg_is_computed. The
+specifies the window over which the avg is computed. The

```

Comment:
It would be better to spell out "avg" (as "average"). Or if we are referring to the AVG syntax, capitalize it.

Reply:
Thanks Dave, your comment has been incorporated :)